pkg/identityserver/mock/gateway_registry.go

```diff
@@ -56,13 +57,27 @@ type mockISGatewayRegistry struct {
 	gatewayRights map[string]authKeyToRights
 
 	registeredGateway *ttnpb.GatewayIdentifiers
+
+	sync.Mutex
```

Comment:
We generally don't prefer this syntax. Please define the mutex variable explicitly. Also in the other mocks.

Reply:
Made it be similar to how it is in other packages, `mu sync.Mutex`.